lib/metrics/context.js

```diff
@@ -218,9 +219,10 @@ module.exports = function (log, config) {
    * @this request
    * @param {String} flowCompleteSignal
    */
-  function setFlowCompleteSignal (flowCompleteSignal) {
+  function setFlowCompleteSignal (flowCompleteSignal, flowType) {
     if (this.payload && this.payload.metricsContext) {
       this.payload.metricsContext.flowCompleteSignal = flowCompleteSignal
+      this.payload.metricsContext.flowType = flowType
```

Comment:
This mildly hackish `flowType` property was added so that we can get the `group` right on `complete` events.